x/ethbridge/keeper/metadata.go

```diff
@@ -33,6 +47,48 @@ func (k Keeper) AddTokenMetadata(ctx sdk.Context, metadata types.TokenMetadata)
 	return denomHash
 }
 
+func (k Keeper) AddIBCTokenMetadata(ctx sdk.Context, metadata types.TokenMetadata, cosmosSender sdk.AccAddress) string {
+	logger := k.Logger(ctx)
+	if !IsIBCToken(metadata.Name) {
+		logger.Error("Token is not IBC cannot use this keeper")
```

Comment:
Change error message from: `Token is not IBC cannot use this keeper` to `Token is not IBC, cannot modify metadata manually`